packages.json

```diff
@@ -1625,6 +1625,12 @@
         "url": "https://github.com/DevDu/FastCoding-Xcode-Plugin",
         "description": "FastCoding is a plug-in for the fast generation of getter, setter, and lazy methods. ",
         "screenshot":"https://raw.githubusercontent.com/DevDu/FastCoding-Xcode-Plugin/master/screenshot/screenShot3.png"
+      },
+      { 
+	"name": "RealGroup",
+	"url": "https://github.com/yizhaorong/RealGroup.git",
+	"description": "Create directory for Xcode Group",
+	"screenshot": "https://github.com/yizhaorong/RealGroup/blob/master/demo.gif"
```

Comment:
Please link directly to the image, and not the Github page displaying it.

The correct URL is `https://raw.githubusercontent.com/yizhaorong/RealGroup/master/demo.gif`.

Thanks!